references.bib

```diff
@@ -911,5 +911,23 @@ @Article{Stuke2020
 abstract={Data science and machine learning in materials science require large datasets of technologically relevant molecules or materials. Currently, publicly available molecular datasets with realistic molecular geometries and spectral properties are rare. We here supply a diverse benchmark spectroscopy dataset of 61,489 molecules extracted from organic crystals in the Cambridge Structural Database (CSD), denoted OE62. Molecular equilibrium geometries are reported at the Perdew-Burke-Ernzerhof (PBE) level of density functional theory (DFT) including van der Waals corrections for all 62 k molecules. For these geometries, OE62 supplies total energies and orbital eigenvalues at the PBE and the PBE hybrid (PBE0) functional level of DFT for all 62 k molecules in vacuum as well as at the PBE0 level for a subset of 30,876 molecules in (implicit) water. For 5,239 molecules in vacuum, the dataset provides quasiparticle energies computed with many-body perturbation theory in the G0W0 approximation with a PBE0 starting point (denoted GW5000 in analogy to the GW100 benchmark set (M. van Setten et al. J. Chem. Theory Comput. 12, 5076 (2016))).},
 issn={2052-4463},
 doi={10.1038/s41597-020-0385-y},
-url={https://doi.org/10.1038/s41597-020-0385-y%7D
-}
+url={https://doi.org/10.1038/s41597-020-0385-y%7D}
+}
+
+@book{Bunker2006,
+  title={{Molecular Symmetry and Spectroscopy}},
+  author={Bunker, P.R. and Jensen, P. and National Research Council Canada and National Research Council Canada. Monograph Publishing Program},
+  isbn={9780660196282},
+  series={NRC monograph publishing program},
+  year={2006},
+  publisher={NRC Research Press}
+}
+
+@misc{CCCBDB,
+  author = {Russell Johnson},
+  title = {Sicklist Database},
+  year = {1999},
+  month = {1999-10-01},
+  publisher = {CCCBDB Computational Chemistry Comparison and Benchmark Database Sicklist},
+  language = {en},
+}
```

Comment:
I don't think the [sicklist](https://www.nist.gov/publications/sicklist-database) is the correct reference.

Reply:
Sorry I found this
NIST Computational Chemistry Comparison and Benchmark Database,
NIST Standard Reference Database Number 101
Release 22, May 2022, Editor: Russell D. Johnson III
http://cccbdb.nist.gov/
However not sure how it translates into a bibtex references.

Reply:
I accidently looked at the next CCCDBD link and downloaded the sicklist. 😂